ddtrace/internal/core/__init__.py

```diff
@@ -275,10 +277,7 @@ def root(self):
     def span(self) -> "Span":
         if self._inner_span is None:
```

Comment:
```suggestion
        if self._inner_span is None and tracer is not None:
```

Reply:
Can't do it because we have to return a Span. But we should never use this without a tracer anyway.